src/message.ts

```diff
@@ -21,7 +21,7 @@ export class Message extends Uint8Array {
   toJson(): IMessage {
     try {
       const from = this.subarray(0, 53)
-      const text = this.subarray(53, this.length)
+      const text = this.subarray(54, this.length)
```

Comment:
We use `53` as that's the length of a `HOPR Address`. Any reason you changing `53`? This anyway should probably be inside something like `const/index.ts` as to avoid having to chase this everywhere. Thoughts @nionis ?

Reply:
with `53` I was getting an extra `:` not sure why?

Reply:
@nionis Are we including a colon (`:`) in the messages or is it something from the parsing in the `sendMessage` class?

Reply:
it [looks like](https://github.com/hoprnet/hopr-chat/blob/cb6043145ca59f8f356824fb3124ffabe945ba0b/commands/sendMessage.ts#L55) `hopr-chat` includes `:` between the `hopr address` and the `message` (`<hopr address>:<message>`).

@jjperezaguinaga It makes more sense to me to remove `:` now that we have the `includeRecepient` option, also this is what https://github.com/hoprnet/hopr-webapp-demo expects as well.

Reply:
@nionis So should we remove that in `hopr-chat`? I can create that patch today.

Reply:
@jjperezaguinaga yeah we can remove it
@shresthagrawal hey there! you can set it back to `53`